drivers/adc/ad7124/ad7124.c

```diff
@@ -231,10 +260,15 @@ int32_t ad7124_reset(struct ad7124_dev *dev)
 				 wr_buf,
 				 8);
 
-	/* Wait for the reset to complete */
+	/* This is to ensure the CRC is disabled after reset */
+	dev->use_crc = AD7124_DISABLE_CRC;
```

Comment:
this disable CRC is again unrelated to the commit.

Reply:
will split out into separate pull request to ensure CRC setting is disabled after reset